website/sidebars.js

```diff
@@ -146,7 +152,14 @@ module.exports = {
     {
       type: "category",
       label: "Using dbt Cloud",
+        link: {
+        type: 'generated-index',
```

Comment:
whoaaah - is this new?? Love it - just noting that the styles are borked on this page (though you probably already know that 😄 ) https://deploy-preview-998--docs-getdbt-com.netlify.app/docs/dbt-cloud

Reply:
@drewbanin Yes it's new! Just trying to fix those fonts/colors now.